development/helper-methods.md

```diff
@@ -63,40 +79,41 @@ Of course, you can combine all of the above filters:
 
 ```javascript
 var modules = MM.getModules()
-  .withClass("classname1")
-  .exceptwithClass("classname2")
-  .exceptModule(aModule);
+  .withClass('classname1')
```

Comment:
reviewing this is a pain ... so don't expect I read every diff.

you really want to have `'` instead of `"` and no `;` at the end of the line in the docs and the other way round in the code?

Reply:
now the PR should be more readable, so only linebreaks :-)